### PR TITLE
implement iron-validator-behavior

### PR DESCRIPTION
**Pull request by morethanreal**
@cdata @notwaldorf PTAL

### CODE REVIEW

File: iron-validator-behavior.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      new Polymer.IronMeta({type: this.validatorType, key: this.validatorName, value: this});
+    },
+
+    /**
+     * Implement custom validation logic in this function.
+     *
+     * @method validate
+     */
+    /*
+    validate: function(values) {
```
Comment (cdata) [inline iron-validator-behavior.html:56]: How come this is commented out? Didn't they make it so that we can do abstract methods in behaviors now?

Comment (morethanreal) [inline iron-validator-behavior.html:56]: Oh really? Is that in rc.7?

Comment (cdata) [inline iron-validator-behavior.html:56]: I'm not sure, actually. Maybe not..

**Comment by cdata** [comment]
:+1: LGTM I like this pattern a lot.

File: iron-validator-behavior.html (hunk 1 above)
Comment (morethanreal) [inline iron-validator-behavior.html:56]: Do you know of a issue I could follow?

Comment (cdata) [inline iron-validator-behavior.html:56]: There isn't an issue, but I just confirmed with @sorvell that they shipped the behavior and will stamp an RC today.

**Pull request merged**